mpp/include/mpp_scatter.fh

```diff
@@ -79,85 +76,70 @@ subroutine MPP_SCATTER_PELIST_3D_(is, ie, js, je, nk, pelist, array_seg, input_d
      root_pe = 0
      root_pe_test = -999
    endif
+
 ! need this check in case MPI-rank 0 is a member of the pelist
-   call mpp_max(root_pe_test, pelist)
-   if (root_pe_test.lt.0) call mpp_error(FATAL, &
+   if ( ANY(pelist(:).eq.mpp_pe()) ) then
```

Comment:
why is this if-check needed here.  We should've kicked out any pe's not in the pelist with the check at line 67.